### PR TITLE
Sec 5: text improvement

### PR DESCRIPTION
**Pull request by felicialim**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/622.html" title="Last updated on Jul 17, 2023, 10:01 PM UTC (fe61f61)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/622/f047a0f...fe61f61.html" title="Last updated on Jul 17, 2023, 10:01 PM UTC (fe61f61)">Diff</a>

### CODE REVIEW

**Review by sunghee-hwang — APPROVED** [review]
LGTM

File: index.bs [hunk 1]
```diff
@@ -1718,24 +1718,22 @@ Capabilities of the IA parser, decoder, and processor:
...
 
-## OBU Sequence Order ## {#standalone-obu-sequence-order}
+## IA Sequence ## {#standalone-ia-sequence}
 
-An <dfn noexport>IA Sequence</dfn> is composed of a series of OBUs in the sequence of a set of [=Descriptors=] followed by their associated [=IA Data=]s. 
-
-NOTE: In a typical case, the first [=Descriptors=] of the [=IA Sequence=] are all non-redundant (i.e. [=obu_redundant_copy=] = 0). When two [=IA Sequence=]s are concatenated, every OBU of the first [=Descriptors=] of the second [=IA Sequence=] is markded as non-redundant (i.e. [=obu_redundant_copy=] = 0).
+An <dfn noexport>IA Sequence</dfn> is composed of a series of OBUs in the sequence of a set of [=Descriptors=] followed by their associated [=IA Data=]. 
 
 The [=Descriptors=] MAY additionally be repeated redundantly and as frequently as necessary. In this case, the [=obu_redundant_copy=] field in their OBU headers SHALL be set to 1.
```
Comment (sunghee-hwang) [inline index.bs:1725]:
```suggestion
The [=Descriptors=] MAY additionally be repeated redundantly and as frequently as necessary. In this case, the [=obu_redundant_copy=] field in their OBU headers SHALL be set to 1. Within an [=IA Sequence=], each OBU in the first [=Descriptors=] SHALL be regarded as a non-redundant OBU regardless of the value of its [=obu_redundant_copy=].
```

**Review by sunghee-hwang — COMMENTED** [review]
To address issue #635

File: index.bs [hunk 2]
```diff
@@ -1733,7 +1733,7 @@ The figure below shows an example of an [=IA Sequence=].
 A set of [=Descriptors=] SHALL be placed in the following order regardless of where they appear in the bitstream:
 
 1. One [=IA Sequence Header OBU=]
-2. All [=Codec Config OBU=]s
+2. One [=Codec Config OBU=]
```
Comment (felicialim) [inline index.bs:1736]: This is fine for this version, but if future profiles allow multiple codecs, will this be too restrictive?

Comment (sunghee-hwang) [reply index.bs:1736]: Profiles restrict the number of Codec Config OBU. So, we can revert it.

File: index.bs [hunk 3]
```diff
@@ -1718,24 +1718,22 @@ Capabilities of the IA parser, decoder, and processor:
...
 <center><img src="images/IA sequence.png" style="width:100%; height:auto;"></center>
-<center><figcaption>Example of Immersive Audio Sequence</figcaption></center>
+<center><figcaption>Example of an Immersive Audio Sequence</figcaption></center>
 
 ### Descriptor OBUs ### {#standalone-descriptor-obus}
 A set of [=Descriptors=] SHALL be placed in the following order regardless of where they appear in the bitstream:
 
 1. One [=IA Sequence Header OBU=]
-2. All [=Codec Config OBU=]s
+2. One [=Codec Config OBU=]
```
Comment (sunghee-hwang) [inline index.bs:1736]:
```suggestion
2. All [=Codec Config OBU=]s
```

**Review by sunghee-hwang — COMMENTED** [review]
To revert it

**Pull request merged**